### ISSUE TITLE
[Bug] Bis issues after delete case (from alert)

### ISSUE THREAD

**Issue by KRUXLEX** [TheHive-Project/TheHive#2296]
### Request Type
Bug

### Problem Description
We create case from alert and we was working on it. But some operator was miss click and delete case. Now we got a big problem, because:
1. We can add again the same alert
2. We have a problem with observable (**THIS IS PRIORITY!!!**), after deletion some observable stay in database and we can't, delete it via python api, we got a 404 error:  
![image](https://user-images.githubusercontent.com/22861005/146762408-87043b01-6b4c-4725-8e34-3f983ef20448.png)

Or using web panel, because: we can't delete from search panel, any other case haven't this observable and WE CAN'T ADD THIS OBSERVABLE TO ANY OTHER CASE (404 ERROR)
![image](https://user-images.githubusercontent.com/22861005/146762363-6b1ac154-f59b-4bbb-a255-0aff74d40534.png)

### Steps to Reproduce
1. Create case from alert
1. Work on it (add more observable - it must total new, can\t exist in other cases), do task
1. Delete case
1. Try to add the same observable to another case

### Possible Solutions
Don't know?

### Complementary information
What you need? P;ease for very fast fix.

**Comment by mieczkowski** [comment]
It would be great to have some database tool for Hive4. In Hive3 we had everything in elasticsearch and some "manual actions" was easy to perform. In hive4 it is not possible due to some custom graph database build on top of cassandra...

**Comment by priamai** [comment]
Gosh this is terrible! Can you confirm your Hive version and what data storage are you using? 
I am thinking if using the Janus backend we could try to write a utility to manually remove the entity link.

**Comment by priamai** [comment]
> It would be great to have some database tool for Hive4. In Hive3 we had everything in elasticsearch and some "manual actions" was easy to perform. In hive4 it is not possible due to some custom graph database build on top of cassandra...

Yes I used to edit the documents directly from ElasticSearch because the API was limiting.

**Comment by KRUXLEX** [comment]
> Gosh this is terrible! Can you confirm your Hive version and what data storage are you using? I am thinking if using the Janus backend we could try to write a utility to manually remove the entity link.

API: TheHive 4.1.15-1 
Backend is cassandra
